setup/native-script.ps1

```diff
@@ -1,30 +1,76 @@
 
 # A PowerShell script to set up Windows machine for NativeScript development
-# To run it against the PRODUCTION branch (only one supported with self-elevation) use
-# @powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))" 
+# NOTE: The scripts requires at least a version 4.0 .NET framework installed
+# To run it inside a COMMAND PROMPT against the production branch (only one supported with self-elevation) use
+# @powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
+# To run it inside a WINDOWS POWERSHELL console against the production branch (only one supported with self-elevation) use
+# start-process -FilePath PowerShell.exe -NoNewWindow -Wait -ArgumentList "-NoProfile -ExecutionPolicy Bypass -Command iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))"
```

Comment:
Well, you are inside a powershell, why are you running another one? What about simply executing:

```
iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/NativeScript/nativescript-cli/production/setup/native-script.ps1'))
```
?